temporalcli/commands.gen.go

```diff
@@ -1314,6 +1315,7 @@ func NewTemporalServerStartDevCommand(cctx *CommandContext, parent *TemporalServ
 	s.Command.Flags().BoolVar(&s.Headless, "headless", false, "Disable the Web UI.")
 	s.Command.Flags().StringVar(&s.Ip, "ip", "localhost", "IP address bound to the frontend Service.")
 	s.Command.Flags().StringVar(&s.UiIp, "ui-ip", "", "IP address bound to the WebUI. Default is same as '--ip' value.")
+	s.Command.Flags().StringVar(&s.PublicPath, "ui-public-path", "", "The public path for the Web UI.")
```

Comment:
@ggrebert - you have to regen from command file since your last change: `go run ./temporalcli/internal/cmd/gen-commands`